src/user/entities/user.entity.ts

```diff
@@ -0,0 +1,55 @@
+import {
+    Entity,
+    Column,
+    PrimaryGeneratedColumn,
+    OneToOne,
+    OneToMany,
+} from 'typeorm';
+import { Authentication } from '../../authentication/entities/authentication.entity';
+import { Message } from '../../message/entities/message.entity';
+import { GroupMember } from '../../group-member/entities/group-member.entity';
+import { MessageReaction } from '../../message-reaction/entities/message-reaction.entity';
+import { UserWeeklyTarget } from '../../user-weekly-target/entities/user-weekly-target.entity';
+import { Group } from '../../group/entities/group.entity';
+
+@Entity()
+export class User {
+    @PrimaryGeneratedColumn()
+    user_id: number;
```

Comment:
I think we should use `bigint` instead of `number` for the `user_id`,  to ensure the compatibility with FIDO2/Webauthn specs. @SturmDerLiebe what do you think?

Reply:
Yeah, TypeOrm is a bit picky here. `number` gets returned as a TypeScript `number` while `bigint` gets returned as a TS `string` as far as I remember.

> Note about bigint type: bigint column type, used in SQL databases, doesn't fit into the regular number type and maps property to a string instead.

That basically means we should explicitly use
```suggestion
    @PrimaryGeneratedColumn('bigint')
    user_id: string;
```
if possible. @cagataycode I think on your branch we are only using bigint as a TS type. If possible let's change that too.

Reply:
> Yeah, TypeOrm is a bit picky here. `number` gets returned as a TypeScript `number` while `bigint` gets returned as a TS `string` as far as I remember.
> 
> > Note about bigint type: bigint column type, used in SQL databases, doesn't fit into the regular number type and maps property to a string instead.
> 
> That basically means we should explicitly use
> 
> https://typeorm.io/entities#column-types
> 
> if possible. @cagataycode I think on your branch we are only using bigint as a TS type. If possible let's change that too.

`'bigint'` unfortunately isn't accepted as a valid type. We would have to use `type: 'bigint'` or `uuid` instead. What do you think?

Reply:
True, the `@PrimaryGeneratedColumn` only accepts an options object as an argument. Then let's add this.

```suggestion
    @PrimaryGeneratedColumn({type: 'bigint'})
    user_id: string;
```

Reply:
Done!